tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot

```diff
@@ -1,23 +1,60 @@
 *** Settings ***
 Resource        ../../../Resources/Page/LoginPage.robot
 Resource        ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../../Resources/Page/HybridCloudConsole/HCCLogin.robot
 Library         SeleniumLibrary
+Library         OpenShiftCLI
 Suite Setup     Kafka Suite Setup
 Suite Teardown  Kafka Suite Teardown
+Test Setup      Kafka Test Setup
+
+*** Variables ***
+${rhosak_real_appname}=  rhosak
+${rhosak_displayed_appname}=  OpenShift Streams for Apache Kafka
 
 *** Test Cases ***
 Verify RHOSAK Is Available In RHODS Dashboard Explore Page
   [Tags]  ODS-258  Smoke  Sanity
```

Comment:
I'd use the same formatting used for the other TC, i.e.
```
[Tags]  Smoke  Sanity
...         ODS-258
```

Reply:
Yes, I will change it

Reply:
Ok